nanoleaf_poly.py

```diff
@@ -52,57 +53,57 @@ def __init__(self, polyglot, primary, address, name):
 
         polyglot.ready()
         polyglot.addNode(self)
-        
+
     def customDataHandler(self, data):
         self.CustomData.load(data)
 
     def parameterHandler(self, params):
         self.poly.Notices.clear()
 
         # Get and set IP
-        if 'ip' in params and params['ip'] is not "":
```

Comment:
This prevents sever from starting up correctly. I will revert formatting changes though. Have yapf formatting in workspace. Forgot to turn it off.